src/test/java/no/hvl/dat250/h2020/group5/repostories/PollRepositoryTest.java

```diff
@@ -99,4 +99,20 @@ public void shouldReturnPublicPollsTest() {
     Assertions.assertEquals(
         1, pollRepository.findAllByVisibilityType(PollVisibilityType.PRIVATE).size());
   }
+
+  @Test
+  public void shouldSaveUserWhenSavingPoll() {
+    User user = new User().userName("Test");
+    Poll poll = new Poll();
+    poll.addOwner(user);
+
+    Poll savedPoll = pollRepository.save(poll);
+
+    Optional<User> findSavedUser = userRepository.findByUsername("Test");
+
+    Assertions.assertTrue(findSavedUser.isPresent());
+    Assertions.assertNotNull(findSavedUser.get().getId());
+    Assertions.assertEquals(savedPoll, findSavedUser.get().getUserPolls().get(0));
+    Assertions.assertEquals(savedPoll.getPollOwner(), findSavedUser.get());
```

Comment:
Bra! Lager du en test for owner.addPoll(poll) også? 👍

Reply:
Ja, lager en egen userRepository test nå :)